docs/authoring/servertaskauthoring.md

```diff
@@ -0,0 +1,77 @@
+# Server Task Authoring
```

Comment:
This repo is for the open source in the box tasks. Most task author guidance is on the task lib repo. Would that be a better place for this document?

Reply:
Agreed.  We have a node and powershell folder there.  Create a server one.

Reply:
All the server tasks are in this repo and there is nothing in task-lib for server tasks. So why do you want to route users to task-lib for server tasks?